pkg/acsengine/const.go

```diff
@@ -31,11 +31,10 @@ const (
 	// DefaultKubernetesFirstConsecutiveStaticIPOffset specifies the IP address offset of master 0
 	// when VNET integration is enabled.
 	DefaultKubernetesFirstConsecutiveStaticIPOffset = 5
-	// DefaultAgentIPAddressCount is the default number of IP addresses per network interface on agents
-	DefaultAgentIPAddressCount = 1
-	// DefaultAgentMultiIPAddressCount is the default number of IP addresses per network interface on agents,
-	// when VNET integration is enabled. It can be overridden per pool by setting the pool's IPAdddressCount property.
-	DefaultAgentMultiIPAddressCount = 128
+	// DefaultKubernetesMaxPods is the maximum number of pods to run on a node.
+	DefaultKubernetesMaxPods = 110
```

Comment:
Is this value consisent across all acs-engine supported k8s versions? (`1.5`, `1.6`, and `1.7`)